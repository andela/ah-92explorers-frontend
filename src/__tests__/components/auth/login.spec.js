import React from 'react';
import { shallow } from 'enzyme';
import LoginComponent from '../../../components/Auth/Login.jsx';

describe('<Login />', () => {
  let wrapper;

  const props = {
    login: jest.fn(),
    email: '',
    password: '',
    errors: {},
  };

  beforeEach(() => {
    wrapper = shallow(<LoginComponent {...props} />);
  });

  it('should render Login component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render two <Input /> elements', () => {
    expect(wrapper.find('Input')).toHaveLength(2);
  });
});
