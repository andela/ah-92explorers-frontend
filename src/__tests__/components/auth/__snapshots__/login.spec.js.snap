// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should render Login component 1`] = `
<div>
  <form
    className="form mb-5"
    id="loginForm"
  >
    <Input
      label="Email"
      name="email"
      placeholder="Email"
      type="text"
      value=""
    />
    <Input
      label="Password"
      name="password"
      placeholder="Password"
      type="password"
      value=""
    />
    <div
      className="form-group mb-4"
    >
      <span
        className="small"
      >
        Donâ€™t have an account?
         
        <Link
          to="/signup"
        >
          Join
        </Link>
      </span>
    </div>
    <div
      className="form-group"
    >
      <input
        className="btn btn-primary btn-block"
        type="submit"
        value="Log in with Email"
      />
    </div>
  </form>
</div>
`;
